src/lib/components/annotation-box/custodial-warning.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+  import AnnotationBox from './annotation-box.svelte';
+  import ThumbsUpIcon from '$lib/components/icons/ThumbsUp.svelte';
+  import Button from '../button/button.svelte';
+  import dismissablesStore from '$lib/stores/dismissables/dismissables.store';
+  import { browser } from '$app/environment';
+  import { slide } from 'svelte/transition';
+
+  export let dismissableId: string = 'custodial';
+
+  let dismissed = browser ? dismissablesStore.isDismissed(dismissableId) : false;
+
+  async function handleDismiss() {
+    dismissablesStore.dismiss(dismissableId);
+  }
+
+  dismissablesStore.subscribe(() => {
+    dismissed = dismissablesStore.isDismissed(dismissableId);
+  });
```

Comment:
no need for manually subbing to `dismissablesStore`! Should be possible to just do `{#if $dismissablesStore.includes(dismissableId)` on line 22